backbone/src/main/java/org/researchstack/backbone/step/active/recorder/PedometerRecorder.java

```diff
@@ -128,16 +123,18 @@ public void onStepTaken() {
         }
     }
 
+
     @Override
-    public void onSensorChanged(SensorEvent sensorEvent) {
+    public void recordSensorEvent(SensorEvent sensorEvent, JsonObject object) {
         // Step detector is only available for OS kitkat and above
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT &&
-            sensorEvent.sensor.getType() == Sensor.TYPE_STEP_DETECTOR)
+                sensorEvent.sensor.getType() == Sensor.TYPE_STEP_DETECTOR)
         {
             onStepTaken();
         } else if (sensorEvent.sensor.getType() == Sensor.TYPE_ACCELEROMETER) {
             accelerometerStepDetector.processAccelerometerData(sensorEvent);
         }
+        //TODO: fix accelerometer data
```

Comment:
What's wrong with accel data?

Reply:
This was a reminder to change AccelerometerRecorder to use the new format of DeviceMotion

Reply:
I still see this line, is this fixed?